source/agora/consensus/state/Ledger.d

```diff
@@ -899,35 +900,12 @@ public class Ledger
 
     ***************************************************************************/
 
-    protected bool hasMajoritySignature (in BlockHeader header,
-        in ValidatorInfo[] validators) @safe nothrow
+   public bool hasMajoritySignature (in BlockHeader header) @safe nothrow
```

Comment:
`nothrow @nogc` too right ?

Reply:
Actually not easy as `percentage` of `BitMask` is not `@nogc` due to `notSetIndices` using a lambda.